contracts/LockedAccount.sol

```diff
@@ -302,6 +335,8 @@ contract LockedAccount is
         return PENALTY_FRACTION;
     }
 
+    // AUDIT[CHF-132] Rename LockedAccount.balanceOf().
```

Comment:
investor balance here is a tuple... what's wrong with it?

Reply:
Not a big deal, but one of the values is the unlock date. It does not match the "balance" name so much.